File: backend/services/post-ts-koa-mongo-fdm/src/common/api/invalid-title.ts
import { ErrorBody } from "../utils/error";

export const empty: ErrorBody = {
    error: {
        error: "TITLE_EMPTY",
        reason: "Title cannot be empty",
        message: "Title cannot be empty",
    }
};

export const tooShort: ErrorBody = {
    error: {
        error: "TITLE_TOO_SHORT",
        reason: "title is too short",
        message: "title is too short",
    }
};

export const tooLong: ErrorBody = {
    error: {
        error: "TITLE_TOO_LONG",
        reason: "title is too long",
        message: "title is too long",
    }
};

File: backend/services/post-python-fastapi-mongo-fdm/src/api/invalid_title.py
from typing import Final
from utils.error import ErrorBody

empty: Final[ErrorBody] = {
    "error": {
        "error": "TITLE_EMPTY",
        "reason": "Title cannot be empty",
        "message": "Title cannot be empty",
    }
}

tooShort: Final[ErrorBody] = {
    "error": {
        "error": "TITLE_TOO_SHORT",
        "reason": "Title is too short",
        "message": "Title is too short",
    }
}

tooLong: Final[ErrorBody] = {
    "error": {
        "error": "TITLE_TOO_LONG",
        "reason": "Title is too long",
        "message": "Title is too long",
    }
}
